### ISSUE TITLE
[sign_in_with_apple] type 'ASAuthorizationError.Code' has no member 'notInteractive'

### ISSUE THREAD

**Issue by anamba** [aboutyou/dart_packages#236]
The latest version now works with Xcode 13 when building for iOS, but when building for macOS, I get this:

```
/Users/anamba/flutter/.pub-cache/hosted/pub.dartlang.org/sign_in_with_apple-3.1.0/macos/Classes/SignInWithAppleError.swift:112:19: error: type 'ASAuthorizationError.Code' has no member 'notInteractive'
            case .notInteractive:
                 ~^~~~~~~~~~~~~~
Command CompileSwiftSources failed with a nonzero exit code
```

**Comment by HenriBeck** [comment]
Hey, Thanks for the report. I will have a look later today. For now please downgrade XCode and wait until we provide a fix here.

**Comment by knopp** [comment]
`ASAuthorizationErrorNotInteractive` is not present in the SDK shipped with XCode 13 on Big Sur.

**Comment by HenriBeck** [comment]
@anamba I just released version 3.2.0 which should fix the build for macOS applications
